pysingfel/detector/tests/test_autoranging_detector.py

```diff
@@ -70,6 +70,7 @@ def test_add_phase_shift(self):
         pattern = self.det.add_phase_shift(self.pattern_0, self.part_coord_1)
         assert np.allclose(pattern, self.pattern_1)
 
+    @pytest.mark.skipif(six.PY3, reason="Calibration constants require psana3")
```

Comment:
change to psana2

FYI. psana1 has a python3 version. At some point, psana1 and psana2 will be PY3

Reply:
Thanks for that clarification. After importing psana, how does one determine whether it's psana1 or psana2? I didn't see a psana.__version__ or equivalent function call, and psana.__package__ just returns 'psana'.

Reply:
I'm not aware of an official way to test for psana version. Let's assume for now PY2 is psana1 and PY3 is psana2 for now.

Let change to:
"Calibration constants require psana3" -> "Calibration constants require psana2"

Reply:
Sounds good. Typo fixed in updated commit.